noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/AsterixHttpRequest.scala

```diff
@@ -0,0 +1,52 @@
+package edu.uci.ics.cloudberry.noah.feed
+
+import play.api.libs.ws.ning.NingWSClient
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.io.Source
+import java.io.File
+
+object AsterixHttpRequest {
+
+  def createTwitterDS(ddlFilename:String){
+    val f = new File(getClass.getClassLoader.getResource(ddlFilename).getPath)
+    val source = Source.fromFile(f)
+    val lines = try source.mkString finally source.close()
+    val query = "http://kiwi.ics.uci.edu:19002/ddl?ddl=" + lines
+
+    httpRequest(query)
+
+  }
+  def queryDB(dataverse:String, dataset:String){
+    val url = "http://kiwi.ics.uci.edu:19002/query?query=use dataverse "+dataverse+"; for $l in dataset('"+dataset+"') return $l;"
+    httpRequest(url)
+  }
+
+  def insertDB(dataverse:String, dataset:String, record:String){
+    val adm = TagTweetNoGeoTagException.tagOneTweet(record)
+    println("ADM = "+ adm)
+    val url = "http://kiwi.ics.uci.edu:19002/update?statements=use dataverse "+dataverse+"; insert into dataset "+dataset+"("+adm+");"
```

Comment:
here is the problem, the string gives to http url should be encoded. This post will help: http://alvinalexander.com/blog/post/java/how-encode-java-string-send-web-server-safe-url